src/crawlee/http_crawler/_http_parser.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from typing_extensions import override
+
+from crawlee.static_content_crawler._static_content_parser import BlockedInfo, StaticContentParser
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable
+
+    from crawlee.http_clients import HttpResponse
+
+
+class NoParser(StaticContentParser[bytes]):
+    """Dummy parser for backwards compatibility.
+
+    To enable using HttpCrawler without need for additional specific parser.
+    """
+
+    @override
+    async def parse(self, http_response: HttpResponse) -> bytes:
+        return http_response.read()
+
+    @override
+    def is_blocked(self, parsed_content: bytes) -> BlockedInfo:  # Intentional unused argument.
```

Comment:
Could we use the `_` convention instead of a comment explaining that it is intentional? (applies to other occurrences as well)

```suggestion
    def is_blocked(self, _: bytes) -> BlockedInfo:
```

Reply:
Fine by me. Hopefully Pyright won't complain